### PR TITLE
Add git-resolver/PipelineTask example to tests

### PR DESCRIPTION
**Pull request by abayer**
# Changes

This uses `/task/git-clone/0.7/git-clone.yaml` on `main` in the Catalog repo as the remote task.

/kind misc

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by abayer** [comment]
/retest

Hmm. The test passes for me locally - I'm going to rerun it once to see if anything's different.

**Comment by abayer** [comment]
Ok, still failing. Something else is going on, not sure what - I'll get back to this later.

**Comment by abayer** [comment]
/hold

Holding while I do some debug logging.

File: examples/v1beta1/pipelineruns/alpha/git-resolver.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          resolver: git
+          params:
+            - name: url
+              value: https://github.com/tektoncd/catalog.git
+            - name: pathInRepo
+              value: /task/git-clone/0.7/git-clone.yaml
+            - name: revision
+              value: main
+        params:
+        params:
```
Comment (Yongxuanzhang) [inline examples/v1beta1/pipelineruns/alpha/git-resolver.yaml:32]: This is duplicate? And cause err:
```
  31:16     error    empty value in block mapping  (empty-values)
  32:9      error    duplication of key "params" in mapping  (key-duplicates)
```

**Comment by abayer** [comment]
/retest
/hold cancel

Ok, so for some reason, using `test/git-resolver/remote-task.yaml` doesn't work in CI, but using a task from the catalog's git repo does work, so hey, let's just use that instead.

**Review by dibyom — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**